src/03-control-flow.js

```diff
@@ -45,7 +57,15 @@ function findLarger(a, b) {}
  *  findLargerOrTie(0, 0);
  *  //> "tie"
  */
-function findLargerOrTie(a, b) {}
+function findLargerOrTie(a, b) {
```

Comment:
These functions look great but I would challenge you to try to use a ternary to make your file even shorter.